### DISCUSSION TITLE
Unpooling layer❓ [QUESTION]

### DISCUSSION THREAD

**Discussion by KirillKulaev** [e3nn/e3nn#495] [Q&A]
Hello, I am working on material characterization and as part of this work I would benefit from a molecule geometry autoencoder, I have already implemented the geometry encoder using e3nn.nn.models.gate_points_2101.Network, however I am having trouble implementing the unpooling layer. I would be glad if someone could tell me how to make unpooling layer using e3nn

**Comment by mitkotak** [comment]
@ameya98
